components/builder-jobsrv/src/server/log_archiver/s3.rs

```diff
@@ -115,11 +117,11 @@ impl LogArchiver for S3Archiver {
             }
         };
 
-        let body = stream.concat2().wait().unwrap();
+        let bytes = stream.concat2().wait().unwrap();
```

Comment:
It's not clear to me why `unwrap` is safe here. If it is, an `expect` with why it is would be helpful.

Reply:
Changing to an expect